db/migrate/20250319234928_add_type_to_appropriate_bodies.rb

```diff
@@ -5,9 +5,14 @@ def up
 
     add_column :appropriate_bodies, :body_type, :appropriate_body_type, default: 'teaching_school_hub'
 
-    AppropriateBody.find_each do |appropriate_body|
-      appropriate_body.update_column(:body_type, type_from_name(appropriate_body.name))
-    end
+    execute <<-SQL.squish
+      UPDATE appropriate_bodies
+      SET body_type =
+        CASE
+          WHEN name = '#{AppropriateBodies::Search::ISTIP}' THEN 'national'::appropriate_body_type
```

Comment:
I would hard code the right values in the SQL and not reference the code personally

Reply:
Well, it is just a constant and the class doesn't require `AppropriateBody` to be loaded but ok, I will hardcode it.

Reply:
It is referenced in the methods and not sure whether it would be loaded and compiled when that file was loaded gut yes, to be on the safe-side this feels better, thanks.